integration/container/links_linux_test.go

```diff
@@ -0,0 +1,57 @@
+package container
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/integration/util/request"
+	"github.com/docker/docker/pkg/stdcopy"
+	"github.com/gotestyourself/gotestyourself/poll"
+	"github.com/gotestyourself/gotestyourself/skip"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLinksEtcHostsContentMatch(t *testing.T) {
+	hosts, err := ioutil.ReadFile("/etc/hosts")
+	skip.IfCondition(t, os.IsNotExist(err))
```

Comment:
You can use just `skip.If()`, it should accept bool.  I believe `IfCondition` is deprecated in the version we have.

I think this needs to also skip on `SameHostDaemon` otherwise the test would fail because the `/etc/hosts` wont match:

```go
skip.If(t, !testEnv.IsLocalDaemon())
```

Reply:
Thanks @dnephin. Done.